base_transaction_id/static/src/xml/account_reconciliation.xml

```diff
@@ -2,10 +2,24 @@
 
 <templates xml:space="preserve">
 
-  <t t-name="reconciliation.line.mv_line" t-inherit="account.reconciliation.line.mv_line">
-    <xpath expr="//td[@class='cell_label']" position="inside">
-      <t t-if="line.transaction_ref"> (<t t-esc="line.transaction_ref"/>)</t>
-    </xpath>
+  <t t-extend="reconciliation.line.mv_line">
+    <t t-jquery=".cell_label" t-operation="replace">
+      <!-- I am replacing this, as there is no component inside, that I can find with jquery -->
```

Comment:
what do you mean? `t[t-esc="line.ref"]` should do the trick

https://api.jquery.com/attribute-equals-selector/

Reply:
updated